modules/gitops-configuring-tls-for-redis-with-autotls-disabled.adoc

```diff
@@ -0,0 +1,235 @@
+
+.. Search for `argocds.argoproj.io` and click `ArgoCD` custom resource definition (CRD).
+
+.. On the *CustomResourceDefinition details* page, click the *Instances* tab, and then click *Create ArgoCD*.
+
+.. Edit or replace the YAML similar to the following example:
++
+.Example ArgoCD CR with autotls disabled
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd <1>
+  namespace: openshift-gitops <2>
+spec:
+  ha:
+    enabled: true <3>
+----
+<1> The name of the Argo CD instance. 
+<2> The namespace where you want to run the Argo CD instance. 
+<3> The flag value that enables the HA feature. If you do not want to enable HA, do not include this line or set the flag value as `false`.
+
+.. Click *Create*.
+
+.. Verify that the Argo CD pods are ready and running:
++
+[source,terminal]
+----
+$ oc get pods -n <namespace> <1>
+----
+<1> The namespace where the Argo CD instance is running.
++
+.Example output with HA disabled
+[source,terminal]
+----
+NAME                                  READY   STATUS    RESTARTS   AGE
+argocd-application-controller-0       1/1     Running   0          26s
+argocd-redis-84b77d4f58-vp6zm         1/1     Running   0          37s
+argocd-repo-server-5b959b57f4-znxjq   1/1     Running   0          37s
+argocd-server-6b8787d686-wv9zh        1/1     Running   0          37s
+----
++
+[NOTE]
+====
+It can take a few minutes for the output to appear if you have enabled the Argo CD instances with HA configuration.
+====
++
+[NOTE]
+====
+The HA-enabled TLS configuration requires a cluster with at least three worker nodes.
+====
++
+.Example output with HA enabled
+[source,terminal]
+----
+NAME                                       READY   STATUS    RESTARTS   AGE
+argocd-application-controller-0            1/1     Running   0          10m
+argocd-redis-ha-haproxy-669757fdb7-5xg8h   1/1     Running   0          10m
+argocd-redis-ha-server-0                   2/2     Running   0          9m9s
+argocd-redis-ha-server-1                   2/2     Running   0          98s
+argocd-redis-ha-server-2                   2/2     Running   0          53s
+argocd-repo-server-576499d46d-8hgbh        1/1     Running   0          10m
+argocd-server-9486f88b7-dk2ks              1/1     Running   0          10m
+----
+
+. Create a self-signed certificate for the Redis server by using one of the following options depending on your HA configuration:
+
+* For the Argo CD instance with HA disabled, run the following command:
++
+[source,terminal]
+----
+$ openssl req -new -x509 -sha256 \
+  -subj "/C=XX/ST=XX/O=Testing/CN=redis" \
+  -reqexts SAN -extensions SAN \
+  -config <(printf "\n[SAN]\nsubjectAltName=DNS:argocd-redis.<namespace>.svc.cluster.local\n[req]\ndistinguished_name=req") \ <1>
+  -keyout /tmp/redis.key \
+  -out /tmp/redis.crt \
+  -newkey rsa:4096 \
+  -nodes \
+  -sha256 \
+  -days 10
+----
+<1> The namespace where the Argo CD instance is running.
```

Comment:
@ekristova Revert to the original call-out sentences for commands. The original sentence is the method suggested in OCP doc guidelines too. This one is not at all clear to the user when performing the procedure. That is why I did not comment on callouts under commands. This is Global Change.

Only for callout sin examples do you follow this pattern. NOT for commands.

Reply:
I see, thank you for letting me know!